app/controllers/admin/content_controller.rb

```diff
@@ -52,6 +52,25 @@ def destroy
     redirect_to :action => 'index'
   end
 
+  def merge
+    id = params[:id]
+    merge_id = params[:merge_with][:merge_id]
+    if current_user.admin?
```

Comment:
Creating these local variables first is nice, but if the user isn't an admin, they aren't needed to i would suggest moving them into the first conditional